tensorflow_serving/model_servers/tensorflow_model_server_test.py

```diff
@@ -81,14 +97,29 @@ def RunServer(self, port, model_name, model_path, use_saved_model):
     print 'Server started'
     return 'localhost:' + str(port)
 
+  def RunServerWithModelConfigFile(self, port, model_config_file, use_saved_model, pipe=None):
+    """Run tensorflow_model_server using test config."""
+    print 'Starting test server...'
+    command = os.path.join(self.binary_dir, 'tensorflow_model_server')
+    command += ' --port=' + str(port)
+    command += ' --model_config_file=' + model_config_file
+    command += ' --use_saved_model=' + str(use_saved_model).lower()
+    
+    print command
+    self.server_proc = subprocess.Popen(shlex.split(command), stderr=pipe)
+    print 'Server started'
+    return 'localhost:' + str(port)
+
   def VerifyPredictRequest(self,
                            model_server_address,
+                           model_name='default',
+                           y_value=3.0,
```

Comment:
Please omit the default value (3.0) for y_value. The tests would be more readable if it's specified explicitly at each callsite.